### PR TITLE
Adding Back to Top hyperlinks

### PR DESCRIPTION
**Pull request by NirantK**
Adding internal hyperlinks (Back to Top) to make navigation on small screens easier

Raising this as a pull request so that I can hear your thoughts on this @keon. This should make it easier to nav between relevant sections instead of just following sequentially.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -160,17 +163,21 @@ Bayesian, statistics and Linguistics approaches for Natural Language Processing
...
+* <a id="clojure">**Clojure**</a> | [Back to Top](#contents)
   * [Clojure-openNLP](https://github.com/dakrone/clojure-opennlp) - Natural Language Processing in Clojure (opennlp)
   * [Infections-clj](https://github.com/r0man/inflections-clj) - Rails-like inflection library for Clojure and ClojureScript
   * [postagga](https://github.com/turbopape/postagga) - A library to parse natural language in Clojure and ClojureScript
-* <a id="ruby">**Ruby**</a>
+
+* <a id="ruby">**Ruby**</a> | [Back to Top](#contents)
   * Kevin Dias's [A collection of Natural Language Processing (NLP) Ruby libraries, tools and software](https://github.com/diasks2/ruby-nlp)
   * [Practical Natural Language Processing done in Ruby](https://github.com/arbox/nlp-with-ruby)
 * <a id="rust">**Rust**</a>
```
Comment (keon) [inline README.md]: Here

File: README.md [hunk 2]
```diff
@@ -183,6 +190,8 @@ Text embeddings allow deep learning to be effective on smaller datasets. These a
 
 The most popular names in word embeddings are word2vec by Google (Mikolov) and GloVe by Stanford (Pennington, Socher and Manning). fastText seems to be a fairly popular for multi-lingual sub-word embeddings. 
 
+[Back to Top](#contents)
```
Comment (keon) [inline README.md]: Some are in the bottom of the section and some are right below the section title.

File: README.md [hunk 3]
```diff
@@ -183,6 +190,8 @@ Text embeddings allow deep learning to be effective on smaller datasets. These a
...
+
 #### word2vec
 word2vec was introduced by [T. Mikolov](https://scholar.google.com/citations?user=oBu8kMMAAAAJ&hl=en) et al. when he was with Google.
```
Comment (keon) [inline README.md]: Some sections don't have it

File: README.md [hunk 4]
```diff
@@ -324,17 +347,6 @@ Dodge et. al 2015. Tests Memory Networks on 4 tasks including reddit dialog task
...
-* [Natural Language Processing Blog](http://nlpers.blogspot.ch/) by Hal Daumé III
-* [Machine Learning Blog](https://bmcfee.github.io/#home) by Brian McFee
-* Ultimate Guide to [Understand & Implement Natural Language Processing](https://www.analyticsvidhya.com/blog/2017/01/ultimate-guide-to-understand-implement-natural-language-processing-codes-in-python/)
-* [AI Playbook](http://aiplaybook.a16z.com/): An easily understandable introductory website.
-* [Introduction to NLP](https://hackernoon.com/learning-ai-if-you-suck-at-math-p7-the-magic-of-natural-language-processing-f3819a689386)
-* [A Guide to Natural Language Processing](https://tomassetti.me/guide-natural-language-processing/): it explains the most common algorithms for many NLP tasks, from stemming to the automatic creation of summaries.
-
-
 ## Datasets
 * [nlp-datasets](https://github.com/niderhoff/nlp-datasets) great collection of nlp datasets
```
Comment (keon) [inline README.md]: here

**Comment by NirantK** [comment]
Hey @keon , thanks for the review comments. I should've been more careful. 

In commit https://github.com/keon/awesome-nlp/pull/91/commits/674cb484bf30db65616aab1868585e05f537a285 , I have added back to top hyperlinks now to most sections. In case of a section description, it is separated by a `|` and then added `Back to Top` after the end of section description.

**Pull request merged**